lib/push-notifications.js

```diff
@@ -88,10 +89,10 @@ PushNotifications.prototype.send = function (pushId, data, callback) {
 
   // Android GCM
   if (regIdsGCM[0] != undefined) {
-    data = _.extend({
```

Comment:
if you could explain why we need this change. To avoid storing empty object in memory if no android devices? or why